packages/react-components/react-switch/src/stories/SwitchThemed.stories.tsx

```diff
@@ -33,7 +33,7 @@ export const Themed = () => (
         <Switch checked={false} label="Unchecked and enabled" />
         <Switch checked={false} disabled label="Unchecked and disabled" />
       </FluentProvider>
-      <FluentProvider style={rightSectionStyle} theme={webHighContrastTheme}>
+      <FluentProvider style={rightSectionStyle} theme={teamsHighContrastTheme}>
         <Switch checked={false} label="Unchecked and enabled" />
         <Switch checked={false} disabled label="Unchecked and disabled" />
       </FluentProvider>
```

Comment:
Presumably since we're not recommending using a high contrast theme in web, we should entirely remove any stories that use it, right?

Reply:
Let's do that separately. In regular component stories there should not be multiple themes at all. Already discussed `Card` with @andrefcdias, `Switch` is probably the only other one which does that. I will take care of those.

Reply:
Addressed in https://github.com/microsoft/fluentui/pull/23082